### DISCUSSION TITLE
Create a Next.js App Problem

### DISCUSSION THREAD

**Discussion by micquan** [vercel/next.js#39405] [Help]
The have this error, can you help?
michaelquan@michaels-mini nextjs-blog % npm run dev

> dev
> next dev

/var/folders/w9/n63w6lf53t55c2gc321k8mb00000gn/T/dev-3949cee3.sh: line 1: next: command not found
michaelquan@michaels-mini nextjs-blog %

**Comment by Chapullin** [comment]
I solved it installing `npm i next`
